### ISSUE TITLE
Constructor property promotion and EmptyBlock rule

### ISSUE THREAD

**Issue by mfickers** [magento/magento-coding-standard#385]
### Description

Using the new PHP 8.0 feature of constructor property promotion, the constructor function will have an empty block:
```php
public function __construct(
    private RequestInterface $request
) {}
```
This is not allowed with the current ruleset, as it violates the Magento2.CodeAnalysis.EmptyBlock rule:
```
```

**Comment by nuzil** [comment]
As temp workaround, use 
--exclude=Magento2.CodeAnalysis.EmptyBlock

but yes, this issue is valid and have to be fixed

**Comment by FinnReinhardtBsc** [comment]
How can this still be an ongoing issue? 😅
